downloadmanager/src/main/java/cm/aptoide/pt/downloadmanager/AptoideDownloadManager.java

```diff
@@ -218,23 +218,28 @@ private int getStateIfFileExists(Download download) {
   private void removeAppDownloader(String md5) {
     AppDownloader appDownloader = appDownloaderMap.get(md5);
```

Comment:
Should remove the appDownloader from the hashmap too

Reply:
Fixed